src/main/java/com/google/api/gax/grpc/ApiCallSettings.java

```diff
@@ -74,7 +74,20 @@ public final RetrySettings getRetrySettings() {
     return retrySettings;
   }
 
-  public abstract Builder toBuilder();
+  /**
+   * Create a new builder for ApiCallSettings with default settings.
+   */
+  public Builder newBuilder() {
```

Comment:
This should be static. Also, as for the documentation: it shouldn't create a Builder with default settings, it should create an empty Builder.